tests-e2e/cypress/integration/runs/rdp_main_summary_spec.js

```diff
@@ -43,69 +51,97 @@ describe('runs > run details page > summary', () => {
 
-            // * Assert that data has changed
-            cy.findByTestId('rendered-text').contains('This is my new summary');
-        });
+                // * Assert that data has changed
+                cy.findByTestId('rendered-text').contains('This is my new summary');
+            });
 
-        // * Assert last edition date is visible
-        cy.findByTestId('run-summary-section').contains('Last edited');
-    });
+            // * Assert last edition date is visible
+            cy.findByTestId('run-summary-section').contains('Last edited');
+        });
 
-    it('can be canceled', () => {
-        // # Mouseover the summary
-        cy.findByTestId('run-summary-section').trigger('mouseover');
+        it('can be canceled', () => {
+            // # Mouseover the summary
+            cy.findByTestId('run-summary-section').trigger('mouseover');
 
-        cy.findByTestId('run-summary-section').within(() => {
+            cy.findByTestId('run-summary-section').within(() => {
             // # Click the edit icon
-            cy.findByTestId('hover-menu-edit-button').click();
+                cy.findByTestId('hover-menu-edit-button').click();
+
+                // # Write a summary
+                cy.findByTestId('editabletext-markdown-textbox1').clear().type('This is my new summary');
+
+                // # Cancel changes
+                cy.findByText('Cancel').click();
+
+                // * Assert that data has not changed
+                cy.findByTestId('rendered-text').contains('Add a run summary');
+            });
+
+            // * Assert last edition date is not visible
+            cy.findByTestId('run-summary-section').should('not.contain', 'Last edited');
+        });
+    });
 
-            // # Write a summary
-            cy.findByTestId('editabletext-markdown-textbox1').clear().type('This is my new summary');
+    describe('as viewer', () => {
+        beforeEach(() => {
+            cy.apiLogin(testViewerUser).then(() => {
+                cy.visit(`/playbooks/run_details/${testRun.id}`);
+            });
+        });
 
-            // # Cancel changes
-            cy.findByText('Cancel').click();
+        commonTests();
 
-            // * Assert that data has not changed
-            cy.findByTestId('rendered-text').contains('Add a run summary');
+        it('has a placeholder', () => {
+            // * Assert the placeholder content
+            cy.findByTestId('run-summary-section').findByTestId('rendered-text').contains('There\'s no summary');
         });
 
-        // * Assert last edition date is not visible
-        cy.findByTestId('run-summary-section').should('not.contain', 'Last edited');
+        it('can not be edited', () => {
+            // # Mouseover the summary
+            cy.findByTestId('run-summary-section').trigger('mouseover');
+
+            cy.findByTestId('run-summary-section').findByTestId('hover-menu-edit-button').should('not.exist');
```

Comment:
Nit: missing comment
```suggestion

            // * Verify that the edit button is not rendered
            cy.findByTestId('run-summary-section').findByTestId('hover-menu-edit-button').should('not.exist');
```